tests/arch/arm64/arm64_high_addresses/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+project(hello_world_user)
```

Maybe rename it to match the actual test name?

```suggestion
project(arm64_high_addresses)
```